src/components/installer/installer.ts

```diff
@@ -0,0 +1,215 @@
+'use strict';
+
+import * as fs from 'fs';
+import * as http from 'http';
+import mkdirp = require('mkdirp');
+import * as path from 'path';
+import * as request from 'request';
+import * as tar from 'tar';
+import * as tmp from 'tmp';
+import * as vscode from 'vscode';
+import { Shell, Platform } from '../../shell';
+import { Errorable } from '../../wizard';
+import { exec } from 'child_process';
+
+export async function installKubectl(shell: Shell) : Promise<Errorable<void>> {
+    const tool = 'kubectl';
+    const installFolder = getInstallFolder(shell, tool);
+    await mkdir(installFolder);
+    switch (shell.platform()) {
+        case Platform.Windows:
+        {
+            // Instruction say always to use this link for Windows, not to check the version first
+            const kubectlUrl = 'https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/windows/amd64/kubectl.exe';
```

Comment:
It seems this url will just download `v1.9.0`, but if you change it to the latest stable version `v1.9.6`, it will download the latest version.

Reply:
Hmm, I took the URL from the k8s installation documentation.  Agree that latest stable would be better.